### DISCUSSION TITLE
Use `github.com/**` instead of `togithub.com/**`

### DISCUSSION THREAD

**Discussion by qhy040404** [renovatebot/renovate#18494] [Suggest an Idea]
### What would you like Renovate to be able to do?

While using `togithub.com`, GitHub Mobile app will first open a webview to load the website, and then redirect to Github app to open the target repo or issue. 
Maybe we can use `github.com/**` directly to avoid an unnecessary redirection.

### If you have any ideas on how this should be implemented, please tell us here.

Easily replace `togithub` with `github`

[lib/modules/platform/github/massage-markdown-links.ts#16](https://github.com/renovatebot/renovate/blob/c4a3a7ae94b394ecfee24e2b87c951e8abea2609/lib/modules/platform/github/massage-markdown-links.ts#L16)
```typescript
function massageLink(input: string): string {
  return input.replace(regEx(/(?:to)?github\.com/i), 'github.com');
}
```

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by rarkins** [comment]
This is done intentionally to avoid “backlink spam”. We should never link to GitHub.com directly from other public projects or it creates disturbances
